app/services/sufia/admin_set_create_service.rb

```diff
@@ -1,6 +1,21 @@
 module Sufia
   # Creates AdminSets
   class AdminSetCreateService
+    DEFAULT_ID = 'admin_sets/default'.freeze
```

Comment:
Yes, this knowledge lives in two places. I don't love it but I don't want to commit to more refactoring to fix #3098.